source/onboard/sso-gitlab.rst

```diff
@@ -50,7 +50,10 @@ Step 1: Add an OpenID Connect application to your GitLab account
 
    If your GitLab instance is not set up to use SSL, your URIs must begin with ``http://`` instead of ``https://``.
 
-  c. Select the scopes: ``openid``, ``profile``, and ``email``.
+  c. Select scopes.
+  
+     - For Mattermost Team Edition, select "read_user".
```

Comment:
@cwarnermm - I apologize for this possibly naive question but what makes the team's implementation of OpenID different than Enterprise? I _believe_ that `OpenID` scope would be required for all of the versions.

Reply:
@coltoneshaw - Some replies to the original thread suggested that ``OpenID`` scope wasn't working as expected. @sbishel - Can you help clarify, please?

Reply:
Unless this has changed...OpenId is only available via Enterprise. Team Edition should still be supporting GitLab via OAuth. We don't allow Team Edition to have OpenId, because then they can use it to connect to anything.

Reply:
Are we continuing support for OAuth then? I thought it was being deprecated entirely, or already was.

Reply:
I _do_ think if we continue this then we'll want to break the docs apart or add a section since this doc focuses specifically on OpenID.

Reply:
No, we never deprecated OAuth. We only modified the UI and only for Enterprise. OpenID is simply a "generic" OAuth with "well known" endpoints, the backend still supports OAuth.

Reply:
@cwarnermm - We may want to update the docs about OpenID then. It includes the below and doesn't include any steps related to OAuth 2.0 in this documentation.

> OAuth 2.0 is being deprecated and replaced by OpenID Connect. Refer to product documentation to convert your existing OAuth configuration for GitLab to the OpenID Connect standard.

Reply:
Hey! Just found this conversation and i want to support @coltoneshaw statement. In fact i created an issue earlier this week specific for this phrasing: https://github.com/mattermost/docs/issues/5338